packages/web-pkg/src/composables/piniaStores/config/config.ts

```diff
@@ -96,6 +98,10 @@ export const useConfigStore = defineStore('config', () => {
     }
   }
 
+  const loadConfigUpdates = (data: RawConfig['updates']) => {
```

Comment:
Since this is not loading anything I'd choose a setter style name (`setAvailableUpdates`, `setUpdates`, ... ?)

Reply:
I just used the pre existing name for setting the config "loadConfig" for consistency (this method doesn't load anything as well). 

I left another comment, asking if it would make sense to have a new updateStore, so we can be more free with naming and implement an error and loading state, what's your take on that ?